src/Shared/PackageManagers/ProjectManagerFactory.cs

```diff
@@ -8,55 +8,153 @@ namespace Microsoft.CST.OpenSource.PackageManagers
     using System.Linq;
     using System.Net.Http;
 
-    public static class ProjectManagerFactory
+    public class ProjectManagerFactory
     {
         /// <summary>
-        /// Get an appropriate project manager for package given its PackageURL.
+        /// The delegate to use when defining the function to create a project manager with.
+        /// The only runtime parameter we need is the destination directory. Everything else can be defined in the constructor.
         /// </summary>
-        /// <param name="purl">The <see cref="PackageURL"/> for the package to create the project manager for.</param>
-        /// <param name="httpClientFactory"> The <see cref="IHttpClientFactory"/> for the project manager to use for making Http Clients to make web requests.</param>
-        /// <param name="destinationDirectory">The directory to use to store any downloaded packages.</param>
```

Comment:
Restore doc comment

Reply:
I don't think this applies here anymore? I did still update the documentation.

Reply:
It is a different method but the parameter is named the same. The new method doesn't have a comment for the parameter.

https://github.com/microsoft/OSSGadget/blob/71cae70070f34a783dcefea12fdd7a351cfcd60f/src/Shared/PackageManagers/ProjectManagerFactory.cs#L13-L17

Reply:
Ahh, I did add that. Should be good.